Src/FluentAssertions/Common/TypeExtensions.cs

```diff
@@ -118,7 +118,9 @@ internal static IEnumerable<TAttribute> GetMatchingOrInheritedAttributes<TAttrib
             return GetCustomAttributes(type, isMatchingAttributePredicate, inherit: true);
         }
 
+#pragma warning disable AV1564 // Parameter in public or internal member is of type bool or bool?
```

Comment:
In general I would recommend to always comment the REASON for disabling a rule.
Especially here I would like to know the reason